test/gtest/hash.cpp

```diff
@@ -0,0 +1,292 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "lib/hash.h"
+
+#include <unordered_map>
+#include <unordered_set>
+
+#include "gtest/gtest.h"
```

Comment:
Can we use `#include <gtest/gtest.h>` since this is an external third-party dependency? This can be helpful for some tools like `clang-tidy` and `bazel`. I believe Bazel even throws an error if a system include is in quoted brackets. It just currently does not build the test folder.

Reply:
Sure, done!